src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import './untils/variables';
```

Comment:
There is a typo in the import path './untils/variables'. It should likely be './utils/variables'. Please verify and correct the directory name to prevent import errors.